docs/topics/mta-web-configuring-and-running-an-application-analysis.adoc

```diff
@@ -41,11 +41,12 @@ You can analyze more than one application at a time against more than one transf
 * OpenJDK - upgrades to one of the following JDK versions:
 ** OpenJDK 11
 ** OpenJDK 17
+** OpenJDK 21
 * Linux - ensures there are no Microsoft Windows paths hard-coded into your applications
 * Jakarta EE 9 - for migrating from Java EE 8 to Jakarta EE 9
 * Spring Boot on Red Hat Runtimes
 * Open Liberty
-* Camel - for migrating from Apache Camel 2 to Apache Camel 3
+* Camel - for migrating from Apache Camel 2 to Apache Camel 3 and to Apache Camel 4
 * Azure
 ** Azure App Service
```

Comment:
@emarcusRH There is one more target under Azure >> Azure Kubernetes Service
![Screenshot from 2023-12-14 20-09-55](https://github.com/windup/windup-documentation/assets/87791967/c565d357-cae5-49bd-b60e-84e745d76c02)